phpcs.xml.dist

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <ruleset name="WordPress coding standards">
 	<!-- Check for cross-version support for PHP 5.6 and higher. -->
-	<config name="testVersion" value="5.6-"/>
+	<!--<config name="testVersion" value="5.6-"/>-->
```

Comment:
@GaryJones 

Thank you, this is a great catch. However, wouldn't it be beneficial to add the `PHPCompatibilityWP` dependency on the project instead?

Reply:
It would indeed, but that addition of that may not have been desired (for whatever reason), so I was keeping it outside of the scope of this PR. A new PR that adds `PHPCompatibilityWP` ruleset to `composer.json` would also need to edit this file to add the ruleset reference, so could uncomment this config property then.

Reply:
Thanks for the clarification.